test/xds/xds_server_rbac_test.go

```diff
@@ -660,6 +693,17 @@ func (s) TestRBACHTTPFilter(t *testing.T) {
 				}
 				// Toggle RBAC back on for next iterations.
 				envconfig.XDSRBAC = true
+
+				if test.wantAuthzOutcomes != nil {
+					if diff := cmp.Diff(lb.authzDecisionStat, test.wantAuthzOutcomes); diff != "" {
+						t.Errorf("authorization decision do not match\ndiff (-got +want):\n%s", diff)
```

Comment:
Changed them to `Fatalf` - if the authz decisions don't match it doesn't make a lot of sense to try to compare the authz event outcomes because they are very likely nonsensical (because at a higher level the decision was wrong).